.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
   hooks:
     - id: black
       additional_dependencies: ["click==8.0.4"]

```

Comment:
I would say in light of the advice given [here](https://black.readthedocs.io/en/stable/integrations/source_version_control.html), and the fact that we purport to support 3.10, this should probably be set to `python3.10` to ensure our code styling properly reflects any newer syntax we use. I think the danger of using `python3` is that pre-commit will defer to your local python installation, and so it's possible that different users will produce different styling decisions, and it all ultimately gets lefts to whichever python we install in the environment that executes in GitHub CI

Reply:
Were you getting issues when this was set to `python3.9`?

Reply:
Yup, the lab machines at my school use python 3.8.10, so I was getting this error
```
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/lille198/.cache/pypoetry/virtualenvs/hermes-yIgdeKmV-py3.8/bin/python', '-mvirtualenv', '/home/lille198/.cache/pre-commit/repoomnqaz0z/py_env-python3.9', '-p', 'python3.9')
return code: 1
expected return code: 0
stdout:
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.9'
    
stderr: (none)```

Reply:
Ah ok interesting, I guess that makes sense. Ok I take it back this is probably the most sensible way to do this, though curious what affects it will have cross-platform. Let's keep it in